### PR TITLE
Publish 'core/utils/ready_callbacks' module

### PR DESCRIPTION
**Pull request by KonyukhovSergey**
To use it in .NET wrappers instead of `$.ready`

### CODE REVIEW

**Review by AlekseyMartynov — DISMISSED** [review]

File: js/aspnet.js [hunk 1]
```diff
@@ -170,6 +171,14 @@
...
                 }
             }
+        },
+
+        addInitScript: function(initScript) {
+            initScripts.push(initScript);
+        },
+
+        execInitScripts: function() {
+            iteratorUtils.each(initScripts, function() { this($); });
```
Comment (San4es) [inline js/aspnet.js]: Can we empty the `initScripts` array?

**Comment by San4es** [comment]
Guys, 

Let's check this concept:
https://jsfiddle.net/72hz1nsw/

**Comment by AlekseyMartynov** [comment]
I like the concept 👍

**Comment by San4es** [comment]
We have to take into account this nuance:
https://jsfiddle.net/xrmLuq7b/

This may break МVC controls rendering after `DOMContentLoaded` (load view via ajax?).

Also, take a look here:
https://github.com/DevExpress/DevExtreme/blob/19_1/js/core/utils/ready_callbacks.js#L9

**Review by nightskylark — DISMISSED** [review]

**Review by ovchinnikov — DISMISSED** [review]

**Review by San4es — APPROVED** [review]
LGTM

**Pull request merged**